internal/apiserver/visualization/pdf.go

```diff
@@ -2,36 +2,99 @@ package visualization
 
 import (
 	"fmt"
-	"strconv"
+	"strings"
 
 	"github.com/jung-kurt/gofpdf"
-
-	"github.com/slovak-egov/einvoice/internal/entity"
+	"github.com/lestrrat-go/libxml2"
+	"github.com/lestrrat-go/libxml2/clib"
+	"github.com/lestrrat-go/libxml2/types"
 )
 
-func Generate(inv *entity.Invoice) *File {
-	pdf := gofpdf.New("P", "mm", "A4", "")
+var lineHeight float64 = 5
+var tabSize = 4
+var font = "Arial"
+var pageHeight float64
 
-	pdf.AddPage()
+func generateLines(currentNode types.Node, level int, pdf *gofpdf.Fpdf) error {
+	if currentNode.NodeType() == clib.ElementNode {
+		if pdf.GetX() > pageHeight {
+			pdf.AddPage()
+		}
+
+		// write padding
+		for i := 0; i < tabSize*(level-1); i++ {
+			pdf.Write(lineHeight, " ")
+		}
+
+		// write node name
+		nodeNameParts := strings.Split(currentNode.NodeName(), ":")
+		name := nodeNameParts[len(nodeNameParts)-1]
+		// use red color
+		pdf.SetTextColor(186, 24, 24)
+		// use bold
+		pdf.SetFontStyle("B")
+		pdf.Write(lineHeight, name)
+		pdf.SetFontStyle("")
```

Comment:
also add newline above this line, because it logically belongs elsewhere (writing value not writing tag name)
and add comment
`// Reset font and text color for writing tag value`